exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -60,7 +60,7 @@ pub struct JavaRuntimeProxy {
 
 impl JavaRuntimeProxy {
     /// Runtime Identifier
-    pub const RUNTIME_ID: RuntimeIdentifier = RuntimeIdentifier::Java;
```

Comment:
I kinda not understand the purpose of `RuntimeIdentifier` enum in Core now, as we can't use it anyway for implementing `WellKnownRuntime` trait